### PR TITLE
feat: add Korean translation for Responsive Web Design curriculum. 

### PR DESCRIPTION
**Pull request by AlisonYoon**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [ ] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

This is the second PR to deploy the new language.

### CODE REVIEW

File: client/i18n/locales/korean/motivation.json [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+    "그 누구도 멈출 수 없다!",
+    "레벨 업!",
+    "풀파워 모드!",
+    "깔끔히 끝냈어요.",
+    "멋져요!",
+    "최고!",
+    "앵콜!",
+    "이 해피엔딩 영화의 주인공은 당신!",
+    "화이팅! 해야죠!",
+    ""
```
Comment (ilenia-magoni) [inline client/i18n/locales/korean/motivation.json:32]:
```suggestion
    "화이팅! 해야죠!"
```
If you leave an empty string in here there could be no compliment appearing

**Review by majestic-owl448 — APPROVED** [review]
All good but for a small thing 👍

File: client/i18n/locales/korean/motivation.json (hunk 1 above)
Comment (AlisonYoon) [reply client/i18n/locales/korean/motivation.json]: oops my bad, thanks @ilenia-magoni 😄

**Comment by naomi-lgbt** [comment]
Sorry @AlisonYoon, I overlooked something in the workflow somewhere and the tests are failing because of the casing in the file path.

Don't stress - I'll prioritise a fix in the morning. 💜

File: shared/config/i18n.ts [hunk 2]
```diff
@@ -101,7 +106,8 @@ export const LangCodes = {
   [Languages.Japanese]: 'ja',
   [Languages.German]: 'de',
   [Languages.Arabic]: 'ar',
-  [Languages.Swahili]: 'sw'
+  [Languages.Swahili]: 'sw',
+  [Languages.Korean]: 'ko'
 };
 
 /**
```
Comment (naomi-lgbt) [inline shared/config/i18n.ts:113]: Down on line 116, add `Languages.Korean` to the `hiddenLangs` array.

That'll make this PR safe to merge without blocking deployments. We'll unhide the language once the Korean site itself is live.

Comment (AlisonYoon) [reply shared/config/i18n.ts:113]: done 👍🏼 thanks @naomi-lgbt 😄

**Review by naomi-lgbt — APPROVED** [review]
Let's get this in! Woohoo!

**Pull request merged**